test/nextjs/templates.test.ts

```diff
@@ -42,7 +42,9 @@ describe('Next.js code templates', () => {
 
           // Setting this option to true will print useful information to the console while you're setting up Sentry.
           debug: false,
-        });"
+        });
+
+        export const onRouterTransitionStart = Sentry.captureRouterTransitionStart;"
```

Comment:
Is this something that users could already have defined prior to the wizard setting it? Not saying this needs to be handled in this PR but maybe somthing to keep in mind/follow up on(?)

Reply:
Yes, although highly unlikely.